src/_includes/partials/token_table.njk

```diff
@@ -0,0 +1,58 @@
+<table class="table-display-token-{{token}}">
+<tr>
+{% for header in headers %}
+<th>{{ header[1] }}</th>
+{% endfor %}
+</tr>
+{% comment %}
+tokens: all the tokens
+token: key
+filteredTokens: the filtered tokens
+subTokens: part of the filtered tokens
+{% endcomment %}
+{% assign filteredTokens = token | split: "." | dig: tokens['Tokens'] %}
+{% for item in filteredTokens %}
+{% if subTokens %}
+{% unless subTokens contains item[0] %}
+{% continue %}
+{% endunless %}
+{% endif %}
```

Comment:
We can filter out some tokens if we want. For example, we can leave out only "icons" from the font families by passing `subtokens: "heading body monospace"` when calling the partial